src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="index.html" 
+      method="post" 
+      data-qa="big" 
+      class="form big-bar"
+    >
+      <label class="field">
+        <input 
+        type="search" 
+        name="input" 
+        data-qa="keypress" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="input input-big"
+        >
```

Comment:
```suggestion
        <input 
          type="search" 
          name="input" 
          data-qa="keypress" 
          placeholder="Try &quot;Los Angeles&quot;"
          class="input input-big"
        >
```
<img width="1094" alt="Screenshot 2023-01-27 at 10 03 01" src="https://user-images.githubusercontent.com/94289038/215037226-0e47420a-414e-40ec-8e0c-b5213503f53f.png">

Reply:
fix in all places